docs/usage/self-hosted-configuration.md

```diff
@@ -134,6 +134,14 @@ Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer pack
 
 This configuration will be applied after all other environment variables so that it can be used to override defaults.
 
+## dockerCache
+
+When set to 'volume', Renovate creates and mounts a temporary Docker volume that is used as a cache for package manager files.
+When set to 'mount', a temporary directory is created and mounted instead.
```

Comment:
Why were you thinking we need both? I don't think we've described enough why someone would want to use this OR how they would choose which. I'm not sure we need both, but open to it

Reply:
Once I figured out how to mount cache volumes properly, it became clear to me that folder-based mounting implementation is a low-hanging fuit. I agree, we need to point out more clearly it's a non-mainstream option. Basically, it may be used only when volumes are problematic for some reason.